### ISSUE TITLE
Error deleting git directories when running a new project from the start

### ISSUE THREAD

**Issue by goneall** [lfscanning/scaffold#33]
Below is the stack trace:

```
ubuntu@ip-172-31-10-83:/lfScanningS3/scaffold/2021-10$ sc 2021-10 run TEST-SPDX
/lfScanningS3/scaffold
TEST-SPDX/tools-java: cloning git@github.com:spdx/tools-java.git
Traceback (most recent call last):
  File "/home/ubuntu/scaffold-code/scaffold/scaffold.py", line 149, in <module>
    doNextThing(SCAFFOLD_HOME, cfg, fdServer, prj_only, sp_only)
  File "/home/ubuntu/scaffold-code/scaffold/runners.py", line 31, in doNextThing
    retval = doNextThingForProject(scaffold_home, cfg, fdServer, prj, sp_only)
  File "/home/ubuntu/scaffold-code/scaffold/runners.py", line 50, in doNextThingForProject
    retval = doNextThingForSubproject(scaffold_home, cfg, fdServer, prj, sp)
  File "/home/ubuntu/scaffold-code/scaffold/runners.py", line 146, in doNextThingForSubproject
    return doZipRepoCodeForSubproject(cfg, prj, sp)
  File "/home/ubuntu/scaffold-code/scaffold/zipcode.py", line 26, in doZipRepoCodeForSubproject
    shutil.rmtree(dotgit_path)
  File "/usr/lib/python3.8/shutil.py", line 718, in rmtree
    _rmtree_safe_fd(fd, path, onerror)
  File "/usr/lib/python3.8/shutil.py", line 655, in _rmtree_safe_fd
    _rmtree_safe_fd(dirfd, fullname, onerror)
  File "/usr/lib/python3.8/shutil.py", line 659, in _rmtree_safe_fd
    onerror(os.rmdir, fullname, sys.exc_info())
  File "/usr/lib/python3.8/shutil.py", line 657, in _rmtree_safe_fd
    os.rmdir(entry.name, dir_fd=topfd)
OSError: [Errno 39] Directory not empty: 'pack'
```

**Comment by goneall** [comment]
What's interesting is after the run, I can delete the .git directory both from the command line and from python by running the python CI and entering the same rmtree command.

**Comment by goneall** [comment]
Turns out that the files are `.fuse_hidden...` which are files open when files are still being accessed.

This seems to be caused by files left open in Scaffold.

**Comment by goneall** [comment]
lsof shows the following file open by "git" command:
```
git       11515 ubuntu  DEL       REG               0,50                1057 /lf
ScanningS3/scaffold/2021-10/code/TEST-SPDX/tools-java/spdx/tools-java/.git/objec
ts/pack/pack-c5664d5bb91668322daa9460733126edf5646948.pack
```

**Comment by goneall** [comment]
Resolved with PR #34
